azure-simple/packer/variables.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Missing License header

Reply:
Cannot  add the Licence hearder since Json.